pkgs/gittools/gitversion/registry.yaml

```diff
@@ -6,74 +6,19 @@ packages:
     description: From git log to SemVer in no time
     version_constraint: "false"
     version_overrides:
-      - version_constraint: semver("<= 3.0.0-Beta3")
-        no_asset: true
-      - version_constraint: semver("<= 4.0.0")
-      - version_constraint: semver("<= 5.0.0")
-        no_asset: true
-      - version_constraint: Version == "5.0.1"
-      - version_constraint: semver("<= 5.2.4")
-        asset: gitversion-{{.OS}}-{{.Version}}.{{.Format}}
-        format: tar.gz
-        replacements:
-          darwin: osx
-        supported_envs:
-          - darwin
-          - windows
-          - amd64
-      - version_constraint: semver("<= 5.6.0")
-        asset: gitversion-{{.OS}}-{{.Arch}}-{{.Version}}.{{.Format}}
-        format: tar.gz
-        rosetta2: true
-        windows_arm_emulation: true
-        replacements:
-          amd64: x64
-          darwin: osx
-          windows: win
-        supported_envs:
-          - darwin
-          - windows/amd64
-      - version_constraint: semver("<= 5.7.0")
```

Comment:
Please don't remove old versions as much as possible.
I'll take a look if we can support old versions.

Reply:
I agree with you, but from my point of view old versions like (<5.9.0) contain too many issues and depend on dotnet framework

Reply:
Ah, I see. I understood why you removed old versions.
Thank you for your explanation.
Indeed, they were very old so I think there is no problem to remove them.